### ISSUE TITLE
Update data on CoinDesk Crypto-Economics Explorer

### ISSUE THREAD

**Issue by s-ben** [xaur/decred-issues#84]
Per this [CoinDesk article](https://www.coindesk.com/coindesks-crypto-economic-data-is-now-accessible-on-github), CoinDesk is crowdsourcing its data collection on developer metrics for its Crypto-Economics Explorer. For version 1.0, only the single most active repo for each project is analyzed. This method vastly underrepresents Decred (which is presumably just judged on dcrd). Moving forward, projects will be able to suggest tweaks to their methodology. From the article:

> After we launched and heard the criticisms, we set out to implement our full vision and expand the repos and activity on GitHub we tracked. When we started, however, we quickly realized the complexity of this task. What we wanted to do meant tracking thousands of projects and repositories. What we needed was a solution that could scale to the complexity and size of the industry and that could capture developer interest and activity in a blockchain from “head to toe.”

> Our solution is now to go directly to developer communities, enabling the coders powering various blockchains to provide their input on our methodology in a setting that’s most apt given their work. 

Projects are encouraged to submit a PR to their [CEX-Methodology GitHub repo](https://github.com/CoindeskData/CEX-Methodology) to add relevant repos, change weights given to various metrics, or "help inform any larger methodology change". 

As an example, the [latest PR](https://github.com/CoindeskData/CEX-Methodology/pull/4) is from Bitshares asking for a number of repos to be added. 

I would suggest we at the very least compile a list of repos that CoinDesk may consider significant to the project and submit a PR asking for them to be added. Other suggested tweaks to their weights or general methodology could also be considered. 

Some conversation has happened already in the #research channel ([Matrix link](https://matrix.to/#/!vGasNHFXqjoEWUBTIi:decred.org/$154803064011235LiyLt:decred.org?via=decred.org&via=matrix.org&via=zettaport.com)).

**Comment by xeroc** [comment]
> I would suggest we at the very least compile a list of repos that CoinDesk may consider significant to the project and submit a PR asking for them to be added. Other suggested tweaks to their weights or general methodology could also be considered.

This sounds like a good idea. While having worked on the pull request for bitshares, we were wondering as well, which repos might be worth being added and which ones are less interesting. The criteria used in bitshares' case was to pick actively developed and maintained repositories as those show that "things are progressing" ..

**Comment by s-ben** [comment]
Hey. Thanks for sharing. Yeah, we haven't really discussed yet what repos to cover, but presumably actively developed repos of some importance to the project seem like what CoinDesk is after. I'm just glad someone is finally looking beyond a single repo for their metrics.

**Comment by xaur** [comment]
Good stuff here. Dev tracker that tracks multiple repo is (finally) serious, and Decred would benefit from correct appearance on a serious tracker.

**Comment by xaur** [comment]
Any news?

**Comment by s-ben** [comment]
Last commit to the CEX-Methodology [repo](https://github.com/CoindeskData/CEX-Methodology) was 10 months ago. None of the PRs have been addressed. I think they realized how big a task this was, that they couldn't just automagically crowdsource something like this while retaining objectivity, and just gave up.  Best to focus on messari, others. Someone posted this the other day,
https://coincodecap.com/

Looks like they give Decred a fair shake. We're generally in the 10 ten for dev activity.

**Comment by xaur** [comment]
Yeah coincodecap is interesting, looks like they track all repos properly and actively do interesting studies e.g. https://blog.coincodecap.com/dead-coins-on-crypto-exchanges/

**Comment by s-ben** [comment]
CoinDesk [confirms](https://github.com/CoindeskData/CEX-Methodology/pull/3#issuecomment-554078325) they've given up on this 'open sourced' initiative, talks vaguely about new upcoming data projects.


![image](https://user-images.githubusercontent.com/1634777/69000348-f953a280-0882-11ea-8ebe-6f1bd3105a48.png)

**Comment by xaur** [comment]
Welcome to real life - open source/open data doesn't happen by itself and there's no free "crowdsourcing" to "the community". You need dedicated maintainers.

Their comment reads like they are going to review and merge some PRs but then abandon the project.

**Comment by s-ben** [comment]
Right? Looks like they also abandoned this almost immediately. They could have at least let people know in the comments instead of  letting people keep creating PRs with zero communication.
